src/utils/getConfig/getStoredConfig.ts

```diff
@@ -0,0 +1,39 @@
+import tenantConfig from '../../../tenant.config';
```

Comment:
This code and the file https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/utils/storeConfig.ts should be together in one file and not separated. Also we should think about if we really want to store the whole config, but also the single values for country as `countryCode` and currency as `currencyCode` as own values in localstorage.

Reply:
first part done and created a new issue #515 to discuss the second part.